plugin/trino-jmx/src/main/java/io/trino/plugin/jmx/JmxHistoricalData.java

```
@@ -66,17 +88,18 @@ public Set<String> getTables()
     public synchronized void addRow(String tableName, List<Object> row)
     {
         String lowerCaseTableName = tableName.toLowerCase(Locale.ENGLISH);
-        checkArgument(tableData.containsKey(lowerCaseTableName));
-        tableData.get(lowerCaseTableName).add(row);
+        if (tableData.containsKey(lowerCaseTableName)) {
+            tableData.get(lowerCaseTableName).add(row);
+        }
```

Can we revert this ? We should throw an error if the table doesn't exist